Vagrantfile

```diff
@@ -46,16 +46,18 @@ Vagrant.configure("2") do |config|
 
 # Install opam and dependencies for compiling xapi
     sudo apt-get install -y opam m4 libxen-dev
-    opam init -a --compiler=4.02.3 -y
+    opam init -a --compiler=4.04.2 -y
     eval `opam config env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
-    opam install -y cppo_ocamlbuild
-    opam depext -y xapi
+    opam depext -y xapi conduit.0.14.5
+    opam install -y conduit.0.14.5
```

Comment:
Should this be fixed from xs-opam?

Reply:
Yes, I'm not sure atm where to add the constraint.

Reply:
Unfortunately it seems that the main opam overrides xs-opam, so unless we remove the default repository we need that pin

Reply:
Otherwise the issue would have been fixed by https://github.com/xapi-project/xs-opam/pull/135

Reply:
So should we fix it in the main repo as well?

Reply:
Why not?

Reply:
There is actually already an upper bound on `conduit` for `cohttp` in the main repo 🤔  :
```
"conduit" {>= "0.14.0" & <"0.16.0"}
```

Reply:
I think we can merge this nevertheless.